docs/logs/discovery/setup.md

```diff
@@ -0,0 +1,8 @@
+title: Setup logs discovery
+description: Getting started with logs discovery
+
+To setup logs discovery you'll have to deploy the Sematext Agent in your infrastructure. Head to the [installation](/agents/sematext-agent/installation) instructions to proceed with the deployment.
```

Comment:
To setup => To set up
you'll have to => simply
Instead of "Head to the [installation](/agents/sematext-agent/installation) instructions to proceed with the deployment." we can just hyperlink "deploy Sematext Agent" in the previous sentence.

Re what @bsmid said above - yeah, that makes sense.  But could we then maybe fix up that agent installation page just a little bit by adding a sentence or two below the agent install commands saying something about needing to set up tokens and linking to a screen in SC that would show the complete STA instructions?  Something like https://apps.sematext.com/ui/monitoring/23327/settings/instructions/overview